src/studio/src/designer/frontend/ux-editor/components/config/ImageComponent.test.tsx

```diff
@@ -34,12 +33,14 @@ describe('>>> ImageComponent/SourceRow', () => {
       },
     };
```

Comment:
Unable to comment on correct line, but rename test from `>>> ImageComponent/SourceRow`